test/AlgebraicGeometry/ToricVarieties/normal_toric_varieties.jl

```diff
@@ -41,6 +41,26 @@
   @testset "Equality of normal toric varieties" begin
     @test (p2 === f2) == false
     @test p2 === p2
-    @test_throws ErrorException("Equality of normal toric varieties is computationally very demanding. More details in the documentation.") p2 == f2
+    @test p2 != f2
+
+    X = projective_space(NormalToricVariety, 2)
+    X = domain(blow_up(X, [3, 4]))
+    X = domain(blow_up(X, [-2, -3]))
+    Y = weighted_projective_space(NormalToricVariety, [1, 2, 3])
+    Y = domain(blow_up(Y, [-1, -1]))
+    Y = domain(blow_up(Y, [3, 4]))
+    @test X == Y
+
+    Z = projective_space(NormalToricVariety, 2)
+    X = domain(blow_up(Z, [1, 1]))
+    Y = domain(blow_up(Z, [1, 2]))
+    @test X != Y
+
+    H = hirzebruch_surface(NormalToricVariety, 0)
+    P1 = projective_space(NormalToricVariety, 1)
+    ray_generators = [[1, 1], [1, 2]]
+    max_cones = incidence_matrix([[1, 2]])
+    X = normal_toric_variety(max_cones, ray_generators)
+    @test length(Set(hash.([H, P1 * P1, X]))) == 2
```

Comment:
```suggestion
    @test length(Set([H, P1 * P1, X])) == 2
```
The julia Set will use hashes (and fall back to the equality), so better use the objects in the Set, otherwise it will fail the test for a hash collision.